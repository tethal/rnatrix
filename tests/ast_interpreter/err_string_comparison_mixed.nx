fun main() {
    print("abc" < 123);
    return 0;
}