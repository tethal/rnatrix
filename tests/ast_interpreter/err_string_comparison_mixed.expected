<string>:2:17: error: operator < cannot be applied to String and Int
    print("abc" < 123);
                ^
